src/components/facets/facets.vue

```diff
@@ -11,6 +11,32 @@
       </Facet>
     </template>
 
+    <template #categoryPaths="{ facet }">
+      <BaseHeaderTogglePanel class="x-facet">
+        <template #header-content>
+          <span>{{ facet.label }}</span>
+          <span class="x-icon">▾</span>
+        </template>
+        <SelectedFilters :facetId="facet.id" />
+        <AllFilter :facet="facet" />
+        <Filters
+          v-slot="{ filter: hierarchicalFilter }"
+          :filters="facet.filters"
+          :animation="staggeredFadeAndSlide"
+        >
+          <HierarchicalFilter
+            v-slot="{ filter, clickFilter, cssClasses, isDisabled }"
+            :filter="hierarchicalFilter"
+          >
+            <label :class="cssClasses">
+              <input @change="clickFilter" :disabled="isDisabled" type="checkbox" />
+              {{ filter.label }}
+            </label>
+          </HierarchicalFilter>
+        </Filters>
+      </BaseHeaderTogglePanel>
+    </template>
```

Comment:
change this to use the current Facet component as in the rest of the Facets.
Also remove any custom content in the filter, this is out of the scope of this task:

```html
    <template #categoryPaths="{ facet }">
      <Facet :facet="facet">
        <AllFilter :facet="facet" />
        <Filters #default="{ filter }" :filters="facet.filters" :animation="staggeredFadeAndSlide">
          <HierarchicalFilter :filter="filter" />
        </Filters>
      </Facet>
    </template>
```

Reply:
where is this change?

Reply:
I did it, I promise...